app/models/attachment.js

```diff
@@ -219,9 +219,29 @@ export function addModel(dbAdapter) {
       this.fileExtension = supportedImageTypes[this.mimeType]
       this.noThumbnail = '1' // this may be overriden below
 
-      // Store original image size
       let originalImage = promisifyAll(gm(tmpAttachmentFile))
```

Comment:
I do not quite understand what it is necessary to move inside an "else". We need originalImage anyway, with rotation and without it.